pkgs/development/haskell-modules/configuration-lts.nix

```diff
@@ -1783,7 +1783,6 @@ self: super: {
   "ceilometer-common" = dontDistribute super."ceilometer-common";
   "cellrenderer-cairo" = dontDistribute super."cellrenderer-cairo";
   "cerberus" = dontDistribute super."cerberus";
-  "cereal" = doDistribute super."cereal_0_5_2_0";
```

Comment:
Why did you remove this override? https://www.stackage.org/lts-6.9 looks like it still has that cereal version